pkg/api/pod/warnings.go

```diff
@@ -234,19 +234,15 @@ func warningsForPodSpecAndMeta(fieldPath *field.Path, podSpec *api.PodSpec, meta
 		warnings = append(warnings, fmt.Sprintf("%s: fractional byte value %q is invalid, must be an integer", fieldPath.Child("spec", "overhead").Key(string(api.ResourceEphemeralStorage)), value.String()))
 	}
 
-	// use of pod seccomp annotation without accompanying field
```

Comment:
I would limit this change to adjusting the warning message.

If they're setting both the annotation and the field, I still don't think we need to warn them.